library/core/class.format.php

```diff
@@ -1339,7 +1339,38 @@ public static function timespan($timespan) {
+    protected static $_UrlTranslations = [
```

Comment:
This change was just to remove duplicate array keys. I had to format the array on multiple lines while I was at it as per our lint rules.